src/Symfony/Bundle/FrameworkBundle/Tests/Functional/app/AutowiringTypes/bundles.php

```diff
@@ -11,8 +11,10 @@
 
 use Symfony\Bundle\FrameworkBundle\Tests\Functional\Bundle\TestBundle\TestBundle;
 use Symfony\Bundle\FrameworkBundle\FrameworkBundle;
+use Symfony\Bundle\SecurityBundle\SecurityBundle;
 
 return array(
     new FrameworkBundle(),
+    new SecurityBundle(),
```

Comment:
@xabbuh I thought about that but it was simpler to integrate a new test here than duplicating the whole test case on the SecurityBundle. Does it hurt?